### PR TITLE
feat(pricefeeds): Add COMPUSDC-APR-MAR28/USDC

### PR DESCRIPTION
**Pull request by evanmays**
**Motivation**

This price identifier is self referential. It tracks the price of the synthetic that uses it.
UMIP-38 for price identifier spec: https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-38.md

**Summary**

Added config for tracking the Uniswap pool for new synthetic.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [x]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js [hunk 1]
```diff
@@ -340,6 +340,11 @@ const defaultConfigs = {
     uniswapAddress: "0x683ea972ffa19b7bad6d6be0440e0a8465dba71c",
     twapLength: 7200
   },
+  "COMPUSDC-APR-MAR28/USDC": {
+    type: "uniswap",
+    uniswapAddress: "0xd8ecab1d50c3335d01885c17b1ce498105238f24",
+    twapLength: 7200
```
Comment (mrice32) [inline packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js:346]:
```suggestion
    twapLength: 7200,
    poolDecimals: 6,
    priceFeedDecimals: 6
```

**Review by mrice32 — COMMENTED** [review]
Looks great! I think we need to specify the poolDecimals for the uniswap feed to function correctly. @nicholaspai can you check my suggestion? I assume this would've _technically_ worked before since the pool decimals and price feed decimals matched, but I think we should be explicit that the pool will output a 1e6 value and that the price feed should also output a 1e6 value. Thoughts?

File: packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js (hunk 1 above)
Comment (nicholaspai) [reply packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js:346]: I agree with this

Comment (mrice32) [reply packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js:346]: edited to remove the priceFeedDecimals

Comment (evanmays) [reply packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js:346]: Cool, I'll test again

**Comment by evanmays** [comment]
Just did an end to end test. Still works! `poolDecimals` is included now

**Review by nicholaspai — APPROVED** [review]
LGTM

**Review by mrice32 — APPROVED** [review]
LGTM!

**Review by chrismaree — APPROVED** [review]

**Review by smb2796 — APPROVED** [review]

**Pull request merged**